src/ngraph/frontend/onnx_import/op/batch_norm.cpp

```diff
@@ -47,18 +61,46 @@ namespace ngraph
                     // TODO: Implement learning mode support
                     // float momentum{node.get_attribute_value<float>("momentum", 0.9f)};
                     ASSERT_IS_SUPPORTED(node, is_test) << "only 'is_test' mode is supported.";
-                    ASSERT_IS_SUPPORTED(node, spatial) << "only 'spatial' mode is supported.";
 
-                    if (inputs.size() >= 5)
+                    if (spatial == 0)
                     {
-                        mean = inputs.at(3);
-                        var = inputs.at(4);
-                        return {std::make_shared<ngraph::op::BatchNormInference>(
-                            epsilon, scale, bias, x, mean, var)};
+                        auto mean =
+                            legacy_style_broadcast_for_binary_operation(data, inputs.at(3), 1)
```

Comment:
It would make sense to turn this into a helper function `legacy_style_broadcast_to(input, new_shape)`?